README.md

```diff
@@ -36,10 +36,13 @@ The last one can be run by:
   - users in the `quizzo-manage-secrets` group
   - programs on ec2 instances which have been started with the `quizzo-secrets` IAM profile
 
-To start an instance with the quizzo-secrets IAM profile from the CLI:
+To start an EC2 instance with the quizzo-secrets IAM profile from the CLI:
 
   `aws ec2 run-instances ...--iam-instance-profile Name=quizzo-secrets`
 
+To start an ECS cluster with the quizzo IAM profile, select quizzo-secrets-instances from the
+Container Instance IAM Role selection on the Create Cluster screen.
```

Comment:
I'd like to, but it's very involved. When I tried to, I said, "Oh! Now I see why Dave and Brian didn't provide them in the original blog post or CLI."  My guess is most people do it through the AWS Console. Or, they use a CloudFormation template, since there are lots and lots of moving pieces.

If you or @themoxman can figure out a 1-liner to get [this Dockerfile](https://github.com/promptworks/aws-secrets/blob/docker-example/example/Dockerfile) running in ECS, please show the way!